fix typo

metamomo opened the pull request:
Differential Revision: D47568928

awgu reviewed (approved):
LGTM!

wanchaol commented on torch/distributed/_tensor/README.md:
```diff
@@ -139,7 +139,7 @@ sharded_module = distribute_module(MyModule(), mesh, partition_fn=shard_fc)
 ## Compiler and PyTorch DTensor
 
-DTensor provides efficient solutions for cases like Tensor Parallelism. But when using the DTensor's replication in a data parallel fashion, it might become observably slower compared to our existing solutions like DDP/FSDP. This is mainly because mainly because DDP/FSDP have a global view of the entire model architecture, thus could optimize for data parallel specifically, i.e. collective fusion ...
+DTensor provides efficient solutions for cases like Tensor Parallelism. But when using the DTensor's replication in a data parallel fashion, it might become observably slower compared to our existing solutions like DDP/FSDP. This is mainly because DDP/FSDP have a global view of the entire model architecture, thus could optimize for data parallel specifically, i.e. collective fusion and computatio ...
```
Another type: "In contract" -> "In contrast", mind fixing it in this PR?

metamomo committed:
fix typo (#105507)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/105507

Differential Revision: D47568928

Pulled By: metamomo

fbshipit-source-id: 9ac011dc412b064e126b3c9f1ccf4a0a3ddcbcdf
